site/topics/overview/overview.rst

```diff
@@ -97,24 +97,219 @@ Types of Evolutionary Computation
+* The *core* components are discussed below at a high-level
+
+
 Representation
 --------------
 
+* The representation is how the problem is encoded
+* The **genotype** is the encoding
+* The **phenotype** is what the encoding means in the context of the problem
+
+
+.. figure:: ../representation/genotype_phenotype_space.png
+    :width: 500 px
+    :align: center
+
+    Visualization of the genotype and phenotype spaces. In this example, the phenotype space consists of integers while
+    the genotype space encodes integers as unsigned binary numbers.
+
+
+* Candidate solution, phenotype, and individual are words used to describe a possible solution to a problem
+* Genotype and chromosome are words used to describe an encoding of a possible solution to a problem
+
+    * However, chromosomes are themselves candidate solutions
+    * "Candidate solution" is often used as a catch-all term
+
+
+* Locus, position, gene, and allele are words used to describe a part of the chromosome
+
+    * Although, this jargon is not commonly used in practice
+
+
+.. figure:: ../overview/genotype_phenotype_binary.png
+    :width: 500 px
+    :align: center
+
+    Example phenotype and genotype for the unsigned binary number maximization problem discussed previously. The
+    Phenotype is the actual integer and the genotype is the binary string. Here, the binary string is shown as a vector.
+    An locus/position/gene/allele would be a single value within the vector (genotype).
+
+
+* It is often ideal to ensure all possible valid solutions can be represented in the genotype space
+* Constraining the search space by eliminating inadmissible solutions can greatly improve performance
+
+* What the encoding for a given problem should is is not always obvious
+* A clever encoding can drastically improve the results of the algorithm
+* These ideas are discussed further in a future topic
+
+
+Fitness and Fitness Function
+----------------------------
 
-Fitness
--------
+* The fitness is the measure of how *good* a given candidate solution is
+* The fitness function is a mechanism for measuring a given candidate solution's *goodness*
+
+* This is what the population is trying to adapt to
+
+* What the fitness function should be is not always straightforward
+
+    * Like representation, the choice of fitness function can drastically impact the performance of the algorithm
+
+
+* Consider the unsigned binary number problem discussed in a previous topic
+* Two different fitness functions were used
+
+    #. The actual integer value of the unsigned binary number
+    #. The number of ones in the unsigned binary number
+
+
+* Although both fitness functions worked on the same representation, the fitness function impacted the performance
+
+    * It altered how the population traversed the genotype space
 
 
 Population
 ----------
 
+* A population is a collection of chromosomes
+
+    * A multiset
```

Comment:
This feels unnecessary. Multiset is the most generic collection, an unordered collection that accepts duplicates. Saying "collection" is sufficient.